charts/nextcloud/templates/deployment.yaml

```diff
@@ -8,17 +9,17 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/component: app
-  {{- if .Values.deploymentLabels }}
-{{ toYaml .Values.deploymentLabels | indent 4 }}
-  {{- end }}
-  {{- if .Values.deploymentAnnotations }}
+    {{- with .Values.deploymentLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- with .Values.deploymentAnnotations }}
```

Comment:
no `with` is an implicite `if`:
you could also use:
```yaml
{{- with .Values.donotexists }}
enabled: true
value: {{ . }}
{{- else }}
enabled: false
{{- end }}
```

Reply:
https://pkg.go.dev/text/template
> {{with pipeline}} T1 {{else}} T0 {{end}}
>	If the value of the pipeline is empty, dot is unaffected and T0
>	is executed; otherwise, dot is set to the value of the pipeline
>	and T1 is executed.